dashboard/models.py

```diff
@@ -13,6 +13,8 @@
 from django.conf import settings
 from django.core.exceptions import ObjectDoesNotExist
 from django.urls import reverse
+from compositefk.fields import CompositeForeignKey
```

Comment:
You're not using this anymore, right? Let's remove it here and in `requirements.txt` if so.

Reply:
Thanks, no I didn't end up using this will remove!